docs/en/04_Changelogs/4.10.0.md

```diff
@@ -6,6 +6,14 @@
 
 ### Other new features
 
+#### New `SS_TEMP_DIR` environment variable
+
```

Comment:
Seems like we should mention that any existing TEMP_PATH/TEMP_FOLDER environment variables will still be respected (I'm assuming that you can just set those)